qa/tasks/ceph.conf.template

```diff
@@ -69,6 +69,7 @@
 
 	bdev debug aio = true
 	osd debug misdirected ops = true
+        osd mclock profile = high_recovery_ops
```

Comment:
Nit: since the lack of this setting causes "Exiting scrub checking -- not all pgs scrubbed" errors, it would make sense to place it near scrub and recovery options, e.g. after `osd scrub max interval = 600`.  It's certainly not a debug option...